### DISCUSSION TITLE
PCL启动游戏时的正版验证问题

### DISCUSSION THREAD

**Discussion by Everchosen091** [Meloong-Git/PCL#5147] [提问] [answered]
自动更新后，PCL2启动器无法离线启动游戏了，提示正版验证。“你必须先登录正版账号，才能进行离线登录”，现在没法免费游玩MC离线版本了吗？
![image](https://github.com/user-attachments/assets/5d37c560-5e1c-4cc0-a50d-943ef8acad0e)

**Comment by Joker2184** [comment]
没有啊... 这个应该是多语言版本才会有的东西..？

**Comment by tangge233** [comment, marked as answer]
这是 PCL 新加入的限制，在某些情况下不会允许您离线游戏……

<details>

<summary>临时解决方案</summary>

修改系统语言为`简体中文`

</details>

**Reply by Everchosen091** [reply]
感谢！
